src/components/SiteHeader.vue

```diff
@@ -150,23 +117,127 @@
+                  'min-h-[20px]',
+                  'ml-[-10px]',
+                  'mb-[-10px]',
+                  'px-[4px]',
+                  'py-[5px]',
+                  'pointer-events-none',
                   'bg-shade-gray',
                   { 'bg-danger': getNotificationCount },
-                  'rounded-full',
-                  'min-w-[24px]',
-                  'px-[8px]',
-                  'py-[4px]',
                 ]"
               >
-                <div class="text-white text-[10px]">
+                <div class="text-white text-[10px] leading-[1em]">
                   {{ formattedNotificationCount }}
                 </div>
               </div>
             </template>
           </MenuItem>
         </MenuList>
       </Dropdown>
+
+      {{ /* phone version */ }}
+      <ButtonV2
+        v-if="!getAddress"
+        class="laptop:hidden"
+        preset="plain"
+        @click="handleOpenSider"
+      >
+        <IconNav />
+      </ButtonV2>
+      <Identity
+        v-if="getAddress"
+        class="cursor-pointer laptop:!hidden ml-[12px]"
+        :avatar-url="walletUserAvatar"
+        :avatar-size="42"
+        :is-avatar-outlined="isWalletUserCivicLiker"
+        @click="handleOpenSider"
+      />
     </div>
+
+    <SiteMenuSider v-if="isShowSider" @close="isShowSider = false">
+      <ButtonV2
+        v-if="!getAddress"
+        class="w-full"
+        preset="secondary"
+        @click="connectWallet"
+        ><div class="flex gap-[12px]">
+          <IconLogin />
+          {{ $t('header_button_connect_to_wallet') }}
+        </div>
+      </ButtonV2>
+      <div v-else class="w-full">
+        <ul class="w-full text-dark-gray">
+          <MenuItem
+            v-for="(item, i) in mainMenuItems"
+            :key="i"
+            class="flex justify-start w-full"
+            :value="item.value"
+            :label="item.name"
+            label-align="left"
+            @select="handleSelectMenuItem"
+          >
+            <template #label-prepend>
+              <MenuIcon :type="item.value" class="mr-[12px]" />
+            </template>
+            <template
+              v-if="item.value === 'notifications' && getNotificationCount > 0"
+              #label-append
+            >
+              <div
+                :class="[
+                  'flex',
+                  'justify-center',
+                  'items-center',
+                  'rounded-full',
+                  'min-w-[20px]',
+                  'min-h-[20px]',
+                  'ml-[-10px]',
+                  'mb-[-10px]',
+                  'px-[4px]',
+                  'py-[5px]',
+                  'pointer-events-none',
+                  'bg-shade-gray',
+                  { 'bg-danger': getNotificationCount },
+                ]"
+              >
+                <div class="text-white text-[10px] leading-[1em]">
+                  {{ formattedNotificationCount }}
+                </div>
+              </div>
```

Comment:
can be simplify to single div?

Reply:
[🎨 Merge duplicated div](https://github.com/likecoin/liker-land/pull/1724/commits/7d8937652e9f0f86b3e3f1a47caadacc89c83394)